plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
@james-lawrence as that pr is staling, can you merge it as it ? or your suggestion is more than a proposal ?